Mobile/Android/Templates/xxx-share__edit_privileges.md

```diff
@@ -0,0 +1,49 @@
+### Sharing privileges
+
+#### Pr: https://github.com/owncloud/android/pull/xxx
+
+Devices:
+
+Server version:
+
+<br>
+ 
+Test Case ID | Test case     | Steps   | Expected behavior | Result | Related Comments
+|:----:|:------------- |:-------------------|:-------------|-------------:|----------:
+**Menu views**|
+1|New file sharing options| 1. Long press in a file to share with users/groups<br>2. Tap on "add user or group" and add one of them<br>3. Tap on handle privileges|2. A badget is shown to handle privileges<br>3. Two options are displayed: "can share" and "can edit", a "Stop sharing" option, and a "Done" button to confirm. On the top appears the name of user/group to share with
+2|New folder sharing options| 1. Long press in a folder to share with users/groups<br>2. Tap on "add user or group" and add one of them<br>3. Tap on handle privileges<br>4. Enable "can edit" option|2. A badget is shown to handle privileges<br>3. Two options are displayed: "can share" and "can edit", a "Stop sharing" option, and a "Done" button to confirm<br>4. Three more options are displayed: "create", "change", "delete". On the top appears the name of user/group to share with
+**Orientations**|
+3|Portrait| From 1 or 2, set on portrait the view of sharing privileges| The view is correcly displayed
+4|Landscape| From 1 or 2, set on landscape the view of sharing privileges| The view is correcly displayed
+**Privileges**|
+5|Can share|1. Long press to share a file/folder with special characters<br>2. Select a user/group and grant it to share<br>3. Login with the sharee<br>4. Share the file/folder with other users| 3. The sharee can see the file<br>4. File/folder is correctly shared
+6|Can not share|1. Long press to share a file/folder<br>2. Select a user/group but does not grant to share<br>3. Login with the sharee<br>4. Share the file/folder with other users| 3. The sharee can see the file<br>4. The file/folder can not be shared
+7|Can edit file|1. Long press to share a file<br>2. Select a user/group and grant it to edit<br>3. Login with the sharee<br>4. Try to edit the file with an external app| 3. The sharee can see the file<br>4. The file is correctly edited
+8|Can not edit file|1. Long press to share a file<br>2. Select a user/group but does not grant it to edit<br>3. Login with the sharee<br>4. Try to edit the file with an external app| 3. The sharee can see the file<br>4. The file is correctly edited
```

Comment:
Typo here? Edit permission is **denied** but the expected result says "The file is **correctly edited** "